ionoscloud/data_source_autoscaling_group_servers.go

```diff
@@ -0,0 +1,55 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services"
+	autoscalingService "github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/autoscaling"
+)
+
+func dataSourceAutoscalingGroupServers() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceAutoscalingServersRead,
+		Schema: map[string]*schema.Schema{
+			"group_id": {
+				Type:             schema.TypeString,
+				Description:      "Unique identifier for the group",
+				Required:         true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.IsUUID),
+			},
+			"servers": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Description: "Unique identifier for the resource",
+							Computed:    true,
+						},
+					}}},
+		},
+	}
+}
+
+func dataSourceAutoscalingServersRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	client := meta.(services.SdkBundle).AutoscalingClient
+
+	id, idOk := d.GetOk("group_id")
+
+	if !idOk {
+		diags := diag.FromErr(fmt.Errorf("autoscaling group_id has to be provided in order to search for its servers"))
```

Comment:
can return `diag.FromErr` directly without the assignment

Reply:
@cbeti-ionos solved in c9df22e0066ca69a7809de59cd124fe97fec9580